File: AngularSonar/src/app/model/SonarData.ts
export class SonarData {
    depth: string;
    status: string;
    battery: string;
    temperature: string;
    error: string;

    constructor(depth: string, battery: string, temperature: string, error: string) {
        this.depth = depth;
        this.battery = battery;
        this.temperature = temperature;
        this.error = error;
    }
}

File: SonarApp/app/src/main/java/com/golovkobalak/sonarapp/model/SonarData.kt
package com.golovkobalak.sonarapp.model

import com.golovkobalak.sonarapp.MainActivity
import io.realm.RealmObject

open class SonarData(
    var depth: String, var battery: String, var temperature: String,
    var time: String, var altitude: String, var accuracy: String
) : RealmObject() {
    constructor() : this("", "", "", "", "", "")
    var sessionId = MainActivity.SESSION_ID
    var altitudeAccuracy: String? = null
    var heading: String? = null
    var latitude = 0.0
    var longitude = 0.0
    var speed: String? = null

    object Field {
        const val LATITUDE = "latitude"
        const val LONGITUDE = "longitude"
    }
}